.github/workflows/release.yml

```diff
@@ -39,19 +39,48 @@ jobs:
       - name: Tests
         run: dotnet test --framework net7.0
 
+      - name: Get changelog entries
+        id: changelog
+        uses: mindsers/changelog-reader-action@v2
+        with:
+          version: Unreleased
+          path: ./CHANGELOG.md
+
+      - name: Update CHANGELOG file
+        uses: thomaseizinger/keep-a-changelog-new-release@1.2.1
+        with:
+          version: ${{ github.event.inputs.versionIncrement }}
+
+      - name: Set git config
+        run: |
+          git config --local user.email "linkdotnet@action.com"
+          git config --local user.name "LinkDotNet Bot"
+
+      - name: Commit changes and push changes
+        run: |
+          git add CHANGELOG.md
+          git commit -m "Update Changelog.md for ${{github.event.inputs.versionIncrement}} release"
```

Comment:
should this commit message not rather be using conventional commits?
something along the lines of :
`docs: changelog for ${{github.event.inputs.versionIncrement}} release`

Reply:
This is part of the ToDo topic (see draft PR description). I reused the information from this article **[From Zero to Production](https://steven-giesel.com/blogPost/5f9e9f0d-2413-4e4b-8e38-9eebe9503e52)** but did not create the environment for the test yet.